auslib/db.py

```diff
@@ -387,7 +387,10 @@ def select(self, where=None, transaction=None, **kwargs):
         if hasattr(where, "keys"):
             where = [getattr(self, k) == v for k, v in where.iteritems()]
 
-        query = self._selectStatement(where=where, **kwargs)
+        if where and isinstance(where[0], list):
```

Comment:
I'm not sure I like this interface change. It's pretty magical to have `select` behave differently based on the type of the first item of the `where` argument. Per the above, this interface change may not be necessary at all.

Reply:
Thanks, hadn't thought of this before. Going to try using `WHERE IN` instead 👍